website/case-studies/introduction-to-flint-modules.md

```diff
@@ -0,0 +1,383 @@
+
+|![Diagram depicting how modules determine pools in FLINT.](../static/img/pools-module-case-study.png) |
+|:--:|
+| Diagram depicting how modules determine pools in FLINT. |
+
+### Operations
+
+An operation is a process within FLINT that moves carbon stock between pools. Operations can reflect ongoing natural processes, such as growth, or specific events, whether natural or human-induced. Operations allow FLINT to track changes in carbon stock through time, including fluxes into and out of pools. For example, a harvest operation moves plant material to products and debris pools, whereas a wildfire operation moves plant material to debris and atmosphere pools.
+
+### Events
+
+Events are operations that occur intermittently (rather than for every time step in a simulation) resulting in the movement of carbon from one pool to another. Events include natural and anthropogenic events including: fire, harvesting, ploughing, and fertiliser application. These are coded for the FLINT as a module. Here is the relationship between Pools and Operations/ Events.
+
+|![Diagram depicting how FLINT calls modules to move carbon between pools. The first column states Carbon Pools/Time step while the first row states Event operations. For each pixel, FLINT calls modules to move carbon between pools.](../static/img/operations-module-case-study.png) |
+|:--:|
+| Diagram depicting how FLINT calls modules to move carbon between pools. |
+
+### Temporal distribution
+
+The temporal scale in FLINT is referred to as time-steps. Time-steps are lengths of time over which operations are reported. It is only at the end of a time-step that carbon can be moved from one pool to another.
+
+The standard time-step in FLINT is one month. However it can be varied by the user. One month is the recommended time-step for modelling carbon.
+
+### Synchronized events
+
+In some circumstances, modules that are simulated for a particular Simulation Unit are dependent on other Simulation Units. That is, there is an interdependency between the Simulation Units. In these situations it is necessary for there to be a process that synchronizes across the related Simulation Units and this is known as synchronized events. The Synchronized Server in FLINT manages Synchronized events.
+
+### Mass balance
+
+The carbon cycle is based on the transfer of carbon molecules from one pool to another, where the molecules are not created, nor are they destroyed. As such, all outputs must be equal to the inputs. The system must be in balance. This is referred to as mass balance. Overall this is the high level design of FLINT.
+
+|![Diagram depicting the high-level design of FLINT.](../static/img/flint-design-module-case-study.png) |
```

Comment:
Make this its own section, as it is not related to mass balance. It describes the software architecture.